app/src/main/java/org/openobservatory/netprobe/data/TestData.java

```diff
@@ -20,28 +21,28 @@
 public class TestData extends Observable {
     private static final String TAG = "TestData";
 
-    public static ArrayList<NetworkMeasurement> mNetworkMeasurementsRunning = new ArrayList<>();
-    public static ArrayList<NetworkMeasurement> mNetworkMeasurementsFinished = new ArrayList<>();
+    //public static ArrayList<NetworkMeasurement> mNetworkMeasurementsRunning = new ArrayList<>();
+    //public static ArrayList<NetworkMeasurement> mNetworkMeasurementsFinished = new ArrayList<>();
```

Comment:
@lorenzoPrimi if these two arrays are not needed anymore, I'd remove them rather than just commenting 'em out